README.md

```diff
@@ -60,8 +60,6 @@ Here's the current matrix of supported features per platform:
 - `tvosX64`
 - `tvosSimulatorArm64`
 
-²: WebSockets are currently not supported on `js`
```

Comment:
This was already in #3927 but we can probably just drop it and keep this one instead 👍